db/deploy/mutations/save_gis_data.sql

```diff
@@ -0,0 +1,27 @@
+-- Deploy ccbc:mutations/save_gis_data to pg
+
+begin;
+
+create or replace function ccbc_public.save_gis_data(json_data jsonb)
+returns ccbc_public.gis_data
+as $function$
+declare
+  result ccbc_public.gis_data;
+  new_row_id int;
+begin
+  -- using nextval instead of returning id on insert to prevent triggering select RLS,
+  -- which requires the gis_data record
+  new_row_id := nextval(pg_get_serial_sequence('ccbc_public.gis_data','id'));
```

Comment:
Do we need this since we're not doing a many to many? I thought we just used this for when we had to create rows in two tables